src/snowflake/snowpark/functions.py

```diff
@@ -3454,6 +3456,9 @@ def sproc(
             command. The default value is 4 and supported values are from 1 to 99.
             Increasing the number of threads can improve performance when uploading
             large stored procedure files.
+        execute_as: What permissions should the procedure have while executing. This
+            supports caller, or owner for now. See https://docs.snowflake.com/en/sql-reference/stored-procedures-rights.html
```

Comment:
```suggestion
            supports caller, or owner for now. See `owner and caller rights <https://docs.snowflake.com/en/sql-reference/stored-procedures-rights.html>`_
```
and let's have this link for all docstrings having this arg?